homeassistant/components/rachio/binary_sensor.py

```diff
@@ -124,11 +124,15 @@ def _poll_update(self, data=None) -> bool:
             '"%s" reported in unknown state "%s"', self.name, data[KEY_STATUS]
         )
 
-    def _handle_update(self, *args, **kwargs) -> None:
+    @callback
+    def _async_handle_update(self, *args, **kwargs) -> None:
         """Handle an update to the state of this sensor."""
-        if args[0][0][KEY_SUBTYPE] == SUBTYPE_ONLINE:
+        if (
+            args[0][0][KEY_SUBTYPE] == SUBTYPE_ONLINE
+            or args[0][0][KEY_SUBTYPE] == SUBTYPE_COLD_REBOOT
```

Comment:
During testing I discovered that after a power cycle subtype COLD REBOOT is returned instead of ONLINE, which is returned after a connectivity loss.